cf-agent/cf-agent.c

```diff
@@ -301,9 +304,11 @@ static GenericAgentConfig *CheckOpts(int argc, char **argv)
      */
     bool cfruncommand = false;
 
-    while ((c = getopt_long(argc_new, argv_new, "tdvnKIf:D:N:VxMB:b:hlC::E", OPTIONS, NULL)) != EOF)
+    while ((c = getopt_long(argc_new, argv_new, "tdvnKIf:D:N:VxMB:b:hlC::ET::",
```

Comment:
You're right, I wasn't sure which one I wanted. I ended up with optional argument, so I'll change the struct, thanks!